takwimu/templates/takwimu/_includes/footer.html

```diff
@@ -29,7 +29,7 @@ <h3 class="font-weight-bold mb-4">Actionable
               <div class="row">
                 <a href="http://www.africapractice.com/" target="_blank" rel="noopener"
                     class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
-                  <img src="{% static 'img/logos/ap-white.png' %}" class="img-responsive" alt="africapractice">
+                  <img src="{% static 'img/logos/ap-footer-white.png' %}" class="img-responsive" alt="africapractice">
```

Comment:
This should just be "ap-white.png". I don't know why there's a different white logo.

Also, please remove all `img-responsive` classes. We've already catered for this in takwimu.scss

Reply:
There was a change request on the ap logo to look all white, it was initially part white and part grey, hence the ap-footer-white. I'll change the logo to reflect this